pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a display columns wide:
```

Comment:
If you are using a concrete value(10) in the example, the description should also use that. So its better to write - "Output is formatted for a 10 columns wide display"